src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__ddi_dashboard.sql

```diff
@@ -0,0 +1,497 @@
+
+union all
+
+select
+    null as student_number,
+    null as student_name,
+
+    w.academic_year,
+
+    r.home_work_location_powerschool_school_id as schoolid,
+    r.home_work_location_abbreviation as school,
+    r.home_work_location_region as region,
+
+    null as grade_level,
+    null as enroll_status,
+    null as cohort,
+
+    cw.grade_band as school_level,
+
+    null as gender,
+    null as ethnicity,
+
+    w.week_start_monday,
+    w.week_end_sunday,
+    w.date_count as days_in_session,
+    w.quarter as term,
+
+    o.rubric_name as title,
+
+    null as scope,
+    null as subject_area,
+
+    o.observed_at as administered_at,
+
+    null as date_taken,
+    null as module_type,
+    null as module_number,
+    'walkthrough' as response_type,
+    null as response_type_code,
+
+    o.measurement_name as response_type_description,
+    o.strand_name as response_type_root_description,
+    o.observation_score as percent_correct,
+
+    if(o.row_score = 1, true, false) as is_mastery,
+
+    null as performance_band_label_number,
+    null as performance_band_label,
+    null as is_replacement,
+    null as qbl,
+    null as grade_goal,
+    null as school_goal,
+    null as region_goal,
+    null as organization_goal,
+    null as course_section,
+    null as course_name,
+    null as course_number,
+    null as course_credittype,
+
+    r.powerschool_teacher_number as course_teachernumber,
+    r.preferred_name_lastfirst as course_teacher_name,
+
+    null as homeroom_section,
+    null as homeroom_teachernumber,
+    null as homeroom_teacher_name,
+    null as n_lessons_passed_reading,
+    null as n_lessons_passed_math,
+    null as pct_lessons_passed_reading,
+    null as pct_lessons_passed_math,
+    null as time_on_task_reading,
+    null as time_on_task_math,
+    null as nj_student_tier,
+
+    lc.head_of_school_preferred_name_lastfirst as head_of_school,
+
+    null as ml_status,
+    null as status_504,
+    null as self_contained_status,
+    null as iep_status,
+
+    o.observation_id as assessment_id,
+    o.row_score as is_mastery_int,
+
+    null as is_qbl,
+    null as illuminate_overview_link,
+    null as illuminate_response_frequency,
+    null as illuminate_matrix,
+from {{ ref("int_performance_management__observation_details") }} as o
+inner join
+    {{ ref("base_people__staff_roster") }} as r on o.employee_number = r.employee_number
+inner join
+    {{ ref("int_powerschool__calendar_week") }} as w
+    on r.home_work_location_powerschool_school_id = w.schoolid
+    and o.observed_at between w.week_start_monday and w.week_end_sunday
+left join
+    {{ ref("stg_people__location_crosswalk") }} as cw
+    on r.home_work_location_name = cw.name
+left join
+    {{ ref("int_people__leadership_crosswalk") }} as lc
+    on r.home_work_location_powerschool_school_id
+    = lc.home_work_location_powerschool_school_id
+where o.observation_type_abbreviation = 'WT'
```

Comment:
why shoehorn this into this view? might want to just add additional columns to existing observation details tableau view